.github/workflows/ruby_ci.yml

```diff
@@ -0,0 +1,45 @@
+# .github/workflows/ruby_tests.yml
+name: Ruby Lint and Test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    env:
+      DBPORT: ${{ job.services.mysql.ports[3306] }}
```

Comment:
Whats the 3306 in brackets for?

Reply:
It's the port number of the service which you want to map.

Reply:
Oh OK, I was just confused because in the other actions (lighthouse for instance) we make the mapping explicit `3306:3306`.

Reply:
I didn't have much luck with that approach, possibly because my actual test suite isn't containerised as well?  Either that or something else was causing trouble, and this change was just part of trouble-shooting.